web/migrations/0063_merge_20251019_2344.py

```
@@ -0,0 +1,13 @@
+# Generated by Django 5.1.6 on 2025-10-19 23:44
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("web", "0062_update_waitingroom_for_sessions"),
+        ("web", "0062_virtualclassroom_virtualclassroomcustomization_and_more"),
+    ]
```

Ruff (0.14.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)